mubench.pipeline/data/detector_execution.py

```diff
@@ -125,7 +119,7 @@ def _run_mode_detector_argument(self):
     def _get_findings_path(self):
         raise NotImplementedError
 
-    def _get_detector_arguments(self, project_compile: ProjectCompile):
+    def _get_command(self, project_compile: ProjectCompile):
```

Comment:
You're right, I deleted the implementations, but forgot the abstract declaration. Which reminds me that there is an API for abstract base classes in python called ABC [[https://pymotw.com/3/abc/]](https://pymotw.com/3/abc/).